### PR TITLE
Add new case related changing values of NetKVM parameters

### PR DESCRIPTION
**Pull request by kimi1978wy**
1 change values of NetKVM parameters-enable/disable
2 change values of NetKVM parameters-multi option
3 change values of NetKVM parameters-modify value directly

Signed-off-by: Yu Wang <wyu@redhat.com>

id: 1745890, 1745892, 1745893

### CODE REVIEW

File: qemu/tests/change_netkvm_params.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    2) set the netkvm driver parameter value
+    3) get the value and compare to target value
+    4) start ping test to check nic avaliability
+
+    param test: the test object
+    param params: the test params
+    param env: test environment
+    """
+
+    def start_test(param_name, param_value):
```
Comment (lolyu) [inline qemu/tests/change_netkvm_params.py]: better to rename this function something like `netkvm_change_param_value_test` to reflect its functionality.

**Comment by kimi1978wy** [comment]
Modified, pls help to review it again @lolyu , thanks a lot

**Comment by kimi1978wy** [comment]
@pezhang  Could you help to review it ?

**Comment by pezhang** [comment]
LGTM.

Acked by: Pei Zhang pezhang@redhat.com

File: qemu/tests/netkvm_change_param_value_test.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    The rx/tx offload checksum test for windows
+    1) start vm
+    2) set the netkvm driver parameter value
+    3) get the value and compare to target value
+    4) start ping test to check nic avaliability
```
[the remark this answers is not in the document]
Comment (kimi1978wy) [reply qemu/tests/netkvm_change_param_value_test.py]: @pezhang Modified, thanks a lot

**Comment by kimi1978wy** [comment]
Remove param_values_NumberOfHandledRXPacketsInDPC according to product bug
Test result:
 (1/3) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.netkvm_change_param_value_test.enable_disable: PASS (548.85 s)
 (2/3) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.netkvm_change_param_value_test.multi_options: PASS (1764.22 s)
 (3/3) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.netkvm_change_param_value_test.modify_directly: PASS (152.03 s)

**Pull request merged**